### PR TITLE
Remove dead JS code

### PR DESCRIPTION
**Pull request by zbynek**
The `YUI.log` was most likely a typo (was meant to be `YAHOO.log`) and is not needed anyway since the standard `console.log` is used in other places. Checks for existence of `console` can be dropped since they are missing in other places. Some global vars are not needed in anymore (mostly because of Prototype removal).

No tests were added/changed because functionality didn't change.
`geval` code was simplified by removing IE-only hacks, but it remains incompatible with CSP (on purpose)

### Testing done

JS lint passes
### Proposed changelog entries

(skip changelog)

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by NotMyFault — APPROVED** [review]

**Comment by MarkEWaite** [comment]
This seems to have caused a regression when creating a new certificate credential with PKCS#12 type.  @zbynek can you investigate [JENKINS-73166](https://issues.jenkins.io/browse/JENKINS-73166) to confirm that my `git bisect` is correct and to identify a fix?

**Comment by MarkEWaite** [comment]
As far as I can tell, I had to restore the following subset of the commit:

```
 function geval(script) {
-  eval(script);
+  // execScript chokes on "" but eval doesn't, so we need to reject it first.
+  if (script == null || script == "") {
+    return;
+  }
+  // see http://perfectionkills.com/global-eval-what-are-the-options/
+  // note that execScript cannot return value
+  (this.execScript || eval)(script);
 }
```

When I replace the `(this.execScript || eval)(script);` with `eval(script)` it fails.

**Pull request merged**
